src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -431,9 +431,9 @@ internal enum EnabledValue
             ExplicitOptOut
         }
 
-        public static EnabledValue ParseEnableValue(string value)
+        public static EnabledValue ParseEnableValue(string? value)
```

Comment:
Is it worth simplifying this method to always return "enabled" unless the value is false or disabled since its either on (by default) or you have to turn it off?

Reply:
We have behaviour differences between explicit opt-in and implicit opt-in, so no, that won't work. I think for telemetry we want to know the difference between "not set" and "invalid value" for implicit opt-in, so I'm keeping those different values (if we see a large number of customers with an invalid value, we might want to investigate why).